src/redux/books/books.js

```diff
@@ -1,28 +1,38 @@
 const ADD_BOOK = 'book-store/books/ADD_BOOK';
 const REMOVE_BOOK = 'book-store/books/REMOVE_BOOK';
 
-const bookReducer = (state = [], action) => {
+const initialState = [
+  { title: 'Hunger Game', author: 'Suzanne Collins', id: 1 },
+  { title: 'Duke', author: 'Franks Haywards', id: 2 },
+  { title: 'Capital in the 20 century', author: 'Suzanne Collins', id: 3 },
+];
+
+const bookReducer = (state = initialState, action) => {
```

Comment:
Great job @monicah79 🤝

- [ ] As stated in the project requirements, kindly make sure adding and removing books is possible.